lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -673,36 +679,37 @@ def self.get_values(json, category_type, override_price = false)
-      't3.micro' => [ 192, 11800], # EBS Optimized
-      't3.small' => [ 192, 11800], # EBS Optimized
-      't3.medium' => [ 192, 11800], # EBS Optimized
-      't3.large' => [ 256, 15700], # EBS Optimized
-      't3.xlarge' => [256, 15700], # EBS Optimized
-      't3.2xlarge' => [ 256, 15700], # EBS Optimized
+      't3.nano' => [ 192, 11800], # EBS Optimized  peak.30min/24hrs, else [4, 250]
+      't3.micro' => [ 192, 11800], # EBS Optimized  peak.30min/24hrs, else [8, 500]
+      't3.small' => [ 192, 11800], # EBS Optimized  peak.30min/24hrs, else [16, 1000]
+      't3.medium' => [ 192, 11800], # EBS Optimized  peak.30min/24hrs, else [32, 2000]
+      't3.large' => [ 256, 15700], # EBS Optimized  peak.30min/24hrs, else [64, 4000]
+      't3.xlarge' => [256, 15700], # EBS Optimized peak.30min/24hrs, else [64, 4000]
+      't3.2xlarge' => [ 256, 15700], # EBS Optimized peak.30min/24hrs, else [64, 4000]
       'x1.16xlarge' => [ 875, 40000],  # EBSOptimized
-      'x1.32xlarge' => [1750, 80000],  # EBSOptimized
+      'x1.32xlarge' => [ 1750, 80000],  # EBSOptimized
       'x1e.xlarge'   => [  62, 3700],  # EBSOptimized
       'x1e.2xlarge'  => [ 125, 7400],  # EBSOptimized
       'x1e.4xlarge'  => [ 219, 10000], # EBSOptimized
       'x1e.8xlarge'  => [ 437, 20000], # EBSOptimized
       'x1e.16xlarge' => [ 875, 40000], # EBSOptimized
       'x1e.32xlarge' => [1750, 80000], # EBSOptimized
-      'z1d.large' => [ 291, 13333], # EBSOptimized
-      'z1d.xlarge' => [ 291, 13333], # EBSOptimized
+      'z1d.large' => [ 291, 13333], # EBSOptimized  peak.30min/24hrs, else [73, 3333]
+      'z1d.xlarge' => [ 291, 13333], # EBSOptimized  peak.30min/24hrs, else [146, 6667]
       'z1d.2xlarge' => [ 292, 13333], # EBSOptimized
       'z1d.3xlarge' => [ 438, 20000], # EBSOptimized
       'z1d.6xlarge' => [ 875, 40000], # EBSOptimized
       'z1d.12xlarge' => [ 1750, 80000], # EBSOptimized
-      'm5a.large' => [ 265, 16000 ], #EBSOptimized
-      'm5a.xlarge' => [ 265, 16000 ], #EBSOptimized
-      'm5a.2xlarge' => [ 265, 16000 ], #EBSOptimized
+      'z1d.metal' => [ 1750, 80000],
+      'm5a.large' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [60, 3600]
+      'm5a.xlarge' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [100, 6000]
+      'm5a.2xlarge' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [146, 8333]
       'm5a.4xlarge' => [ 265, 16000 ], #EBSOptimized
       'm5a.12xlarge' => [ 675, 30000 ], #EBSOptimized
       'm5a.24xlarge' => [ 1250, 60000 ], #EBSOptimized
-      'r5a.large' => [ 265, 16000 ], #EBSOptimized
-      'r5a.xlarge' => [ 265, 16000 ], #EBSOptimized
-      'r5a.2xlarge' => [ 265, 16000 ], #EBSOptimized
+      'r5a.large' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [60, 3600]
+      'r5a.xlarge' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [100, 6000]
+      'r5a.2xlarge' => [ 265, 16000 ], #EBSOptimized peak.30min/24hrs, else [146, 8333]
```

Comment:
would like to go over these (with you) PER_SEC_CAPACITIES changes, and differences between 'peak' and non-peak #'s;   we might want to confirm clients of these numbers, presumably RightSizing, will work correctly as well.

Reply:
Sounds good. These values were directly pulled from the AWS website; I had noticed that some of our values were off, so literally walked through each entry to confirm we're good and up to date with what they've published.

Reply:
http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSOptimized.html